enos/modules/vault_verify_default_lcq/scripts/smoke-verify-default-lcq.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+function fail() {
+  echo "$1" 1>&2
+  exit 1
+}
+
+[[ -z "$RETRY_INTERVAL" ]] && fail "RETRY_INTERVAL env variable has not been set"
+[[ -z "$TIMEOUT_SECONDS" ]] && fail "TIMEOUT_SECONDS env variable has not been set"
+[[ -z "$VAULT_ADDR" ]] && fail "VAULT_ADDR env variable has not been set"
+[[ -z "$VAULT_TOKEN" ]] && fail "VAULT_TOKEN env variable has not been set"
+
+getMaxLeases() {
+  curl --request GET --header "X-Vault-Token: $VAULT_TOKEN" \
```

Comment:
Would it be possible to use `vault read` for this instead of `curl`? We're hoping to eventually rid the need of curl entirely from the runners. If we absolutely can't then there are other avenues that we can go down, but that's where I would start first.